yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -75,6 +83,49 @@ public JNLPLauncher getJnlpLauncher() {
     }
 
     @DataBoundSetter
+    public void setJnlpOpts(String jnlpOpts) {
+        this.jnlpOpts = jnlpOpts;
+    }
+
+    public String getJnlpOpts() {
+        return jnlpOpts;
+    }
+
+    @DataBoundSetter
+    public void setJenkinsUrl(String jenkinsUrl) {
```

Comment:
Yes. Correct. But in our case. It fails because we cannot use the public jenkins url...